service/controller/v2/resource/drainer/create.go

```diff
@@ -6,7 +6,7 @@ import (
 	"time"
 
 	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
-	"github.com/giantswarm/errors/guest"
+	"github.com/giantswarm/errors/tenant"
```

Comment:
This does not look right on its own.

Reply:
I missed the vendor deps. Did in the PR before but this was not good enough as it seems. Should be fixed now.